beacon_node/http_api/src/lib.rs

```diff
@@ -1715,6 +1718,47 @@ pub fn serve<T: BeaconChainTypes>(
             },
         );
 
+    /*
+     * beacon/blob_sidecars
+     */
+
+    // GET beacon/blob_sidecars/{block_id}
+    let get_blobs = eth_v1
+        .and(warp::path("beacon"))
+        .and(warp::path("blob_sidecars"))
+        .and(block_id_or_err)
+        .and(warp::query::<api_types::BlobIndicesQuery>())
+        .and(warp::path::end())
+        .and(chain_filter.clone())
+        .and(warp::header::optional::<api_types::Accept>("accept"))
+        .and_then(
+            |block_id: BlockId,
+             indices: api_types::BlobIndicesQuery,
+             chain: Arc<BeaconChain<T>>,
+             accept_header: Option<api_types::Accept>| {
+                async move {
```

Comment:
This should use the task executor.

Reply:
updated here: 7547e4611f33018ed4234bd944a1d75facba9e75